staging/src/k8s.io/cloud-provider/options/options.go

```diff
@@ -166,6 +172,16 @@ func (o *CloudControllerManagerOptions) ApplyTo(c *config.Config, userAgent stri
 	if err = o.ServiceController.ApplyTo(&c.ComponentConfig.ServiceController); err != nil {
 		return err
 	}
+	if o.Webhook != nil {
```

Comment:
I don't have a good sense for how this library is used, but if webhook / webhookserving could be nil, need to guard the AddFlags and Validate calls as well

Reply:
done.